cassandra/src/main/java/org/hibernate/ogm/datastore/cassandra/impl/CassandraDatastoreProvider.java

```diff
@@ -119,14 +119,13 @@ public void start() {
 						.withCredentials( config.getUsername(), config.getPassword() )
 						.build();
 
-				Session bootstrapSession = cluster.connect();
-				bootstrapSession.execute(
+				session = cluster.connect();
```

Comment:
I have no experience with Cassandra, but by reading the javadoc it seems that we might be able to go a step further:

the `connect()` here is only needed to run the `CREATE KEYSPACE IF NOT EXISTS`.

I'm not sure if all users will want us to invoke that at each bootstrap, shouldn't it depend on the `hibernate.hbm2ddl.auto` ?

Anyway, that's just a thought for future.. the patch looks like better than the original so applying ;)